dashboard/app/models/user_script.rb

```diff
@@ -33,7 +33,6 @@ def check_completed?
   end
 
   def empty?
-    # a user script is empty if there is no progress
-    started_at.nil? && completed_at.nil? && assigned_at.nil? && last_progress_at.nil?
+    started_at.nil? && assigned_at.nil?
```

Comment:
Agreed. Unfortunately, our codebase uses `empty?` in other contexts quite a bit.